app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -292,6 +296,30 @@ String getTestOwner(String builderName, BuilderType type, String testOwnersConte
         }
         break;
       }
+    case BuilderType.firebaselab:
```

Comment:
not related to this pr, Do we want to file mark flaky pr for firebaselab test?

Reply:
I am not sure about that, but i guess we still want to have ownership for those tests.